Assets/Scripts/Lobby/LobbyManager.cs

```diff
@@ -1,22 +1,49 @@
+using System.Collections;
 using Unity.Services.Lobbies;
+using UnityEngine;
 
 namespace Lobby
 {
-    public class LobbyManager
+    public class LobbyManager : MonoBehaviour
     {
+        private Unity.Services.Lobbies.Models.Lobby CurrentLobby { get; set; }
+        private string lobbyId;
+        
         public Unity.Services.Lobbies.Models.Lobby CreateLobby()
         {
             var task = Lobbies.Instance.CreateLobbyAsync("MyLobby", 4, new CreateLobbyOptions()
             {
                 IsPrivate = true
             });
-            return task.GetAwaiter().GetResult();
+            var lobby = task.GetAwaiter().GetResult();
+            CurrentLobby = lobby;
+            lobbyId = lobby.Id;
+            StartCoroutine(GetLobby(2.5f));
+            return lobby;
         }
 
         public Unity.Services.Lobbies.Models.Lobby JoinLobby(string lobbyCode)
         {
             var task = Lobbies.Instance.JoinLobbyByCodeAsync(lobbyCode);
-            return task.GetAwaiter().GetResult();
+            var lobby = task.GetAwaiter().GetResult();
+            CurrentLobby = lobby;
+            lobbyId = lobby.Id;
+            StartCoroutine(GetLobby(0.1f));
+            return lobby;
+        }
+        
+        private void GetLobby()
+        {
+            var task = Lobbies.Instance.GetLobbyAsync(lobbyId);
+            var lobby = task.GetAwaiter().GetResult();
+            StartCoroutine(GetLobby(0.0001f));
+        }
+        
+        private IEnumerator GetLobby(float delay)
```

Comment:
Our coroutine and our void have the exact same name, which isn't good practice. One of them should be renamed.